### DISCUSSION TITLE
web 1.0.0 compatibility

### DISCUSSION THREAD

**Discussion by Kuhlemann** [Aniketkhote/refreshed#38] [Ideas]
Hi @Aniketkhote ,

first of all thank you for your commitment!

Just a quick question: Are you planning to make refreshed "web 1.0.0"-compatible?
If yes, is this a huger project for you, or is it quite simple to update the necessary dependencies?

Best regards
Kuhlemann

**Comment by Aniketkhote** [comment]
Thank you for your support!

This project is quite complex for me as well, especially with the routing system and undocumented methods. I’m learning and working to fix issues, but it’s challenging. Contributions are highly appreciated and would be a huge help in speeding up progress!
